src/pages/markets/_available-platforms.js

```diff
@@ -31,19 +32,24 @@ const PlatformsContainer = styled(Flex)`
     a:last-child {
         margin-right: 0;
     }
-
-    @media ${device.tabletL} {
-        justify-content: flex-start;
-        margin-top: 3rem;
+    @media ${device.tablet} {
+        width: ${(props) => props.width};
     }
 `
 const StyledText = styled(Text)`
     margin-right: 1.6rem;
+    width: 102px;
```

Comment:
why should we use static width here? it affected on `available on` in the mobile view. please check the design.